doc/content/microreactors/mrad/control_drum_rotation.md

```diff
@@ -0,0 +1,156 @@
+
+## Mesh File
+
+
+In this analysis, a 3D whole-core model with 1/6 symmetry was employed. The **MOOSE Reactor Module** provides several tools for generating finite element meshes, enabling rapid construction of detailed reactor geometry—including pins, assemblies, control drums, and peripheral zones—through operations such as extrusion and rotation.
+
+
+!listing /mrad/mesh/HPMR_OneSixth_finercdrum.i max-height = 10000
+
+
+### Mesh Generation Workflow
+
+The meshing process begins by defining various pin types (moderator pins, heat pipe pins, and fuel pins) using the `PolygonConcentricCircleMeshGenerator`.These pins are then arranged within fuel assemblies using the `PatternedHexMeshGenerator`.To handle external boundaries of the hexagonal assemblies, the `PatternedHexPeripheralModifier` is applied. This class uses `FillBetweenPointVectorsTools` to replace the outermost layer of quadrilateral elements with a transitional layer of triangular elements. This ensures node placement at designated boundary positions, facilitating the stitching of adjacent hexagonal assemblies with differing boundary node counts due to interior pin variations or azimuthal discretization.
+
+### Control Drum Meshing
+
+Control drums were meshed using the `HexagonConcentricCircleAdaptiveBoundaryMeshGenerator`. For drum-containing assemblies, the `num_sectors_per_side` parameter was increased to ensure mesh refinement in the drum region, guaranteeing the drum's front face aligns with element edges at each time step.
+
+
+!media media/mrad/mesh_hpmr_drum.png
+       style=display: block;margin-left:auto;margin-right:auto;width:90%;
+       id=hpmr_mesh_used
+       caption=HP-MR mesh used in the analysis.
+	      
+
+### Reflector and 3D Core Construction
+
+The radial reflector was modeled with the `PeripheralRingMeshGenerator` to generate the outer shield region. A dummy fuel assembly was temporarily added to complete the hexagonal core pattern and later removed using the `BlockDeletionGenerator`.
+Assemblies (including those with control drums) were patterned with the `PatternedHexMeshGenerator`. A 1/6 symmetry model of the HP-MR core was created by trimming the full hexagonal core using the `PlaneDeletionGenerator`.The resulting 2D mesh was extruded into 3D using the `AdvancedExtruderGenerator`, with axial layers defined via the `subdomain_swaps` parameter.
+
+
+## Griffin Neutronics Simulation
+
+### Multi-Group Cross Sections
+
+Multi-group cross sections with 11 energy groups were generated using the **Serpent 2 Monte Carlo code**, based on a parametric grid defined by **Control drum rotation angle** (4 points), **Fuel temperature** (5 points), and **Temperature of moderator, reflector, monolith, and heat pipe** (4 points) 
```

Comment:
```suggestion
Multi-group cross sections with 11 energy groups were generated using the **Serpent 2 Monte Carlo code**, based on a parametric grid defined by **Control drum rotation angle** (4 values), **Fuel temperature** (5 values), and **Temperature of moderator, reflector, monolith, and heat pipe** (4 values), for a total of 80 simulations.
```

Reply:
do we know how converged we are with that?
I remember Brendan Kochunas asking about this at the NEAMS review